### ISSUE TITLE
Cannot Toggle Play/Pause

### ISSUE THREAD

**Issue by BrilliantStanley** [Strixx76/samsungwam#2]
https://github.com/Strixx76/samsungwam/assets/78531020/dc4f27fe-92b1-4b30-9972-6989857638cd

Unable to pause or play the speaker. 
Other controls such as shuffle, repeat, prev, next, and volume worked without problem. 
Model : WAM-3500.

**Comment by Strixx76** [comment]
I don't use DLNA my self, so I have not tested this. It works with TuneIn and Spotify which is the only to services I use. The command are different for controlling playback from DLNA, but that is all handled by the underlying package. There I have used what others have discovered.

If you pause in the multiroom app can you then resume playback from the integration?

**Comment by BrilliantStanley** [comment]
I could not play when the music was paused from the multiroom app. The speakers are only able to browse from DLNA folders and can not be controlled by DLNA server (like HA), hence not showing up as DLNA DMR in HA. My setup is using windows media streaming which is quite easy to setup.

I have also found a new issue where I can start tune in playback from the media browser but the status at HA stayed idle with no playback control. Volume control however worked though.
![image](https://github.com/Strixx76/samsungwam/assets/78531020/6b8f8444-fdf6-4f2b-b10f-51fea2a6c2b5)
![image](https://github.com/Strixx76/samsungwam/assets/78531020/6bf02574-c13c-46ca-ab42-57b937eaffc0)

**Comment by Strixx76** [comment]
Have you tried to reload the integration? I have noticed that the integration can loose connection to the speaker if you have not used it for some days. It seems that is what have happened to you. Then the state stop updates but you can still use some controls.
Next release will check connections status but it requires quit some work, and will not be ready for some time.

**Comment by BrilliantStanley** [comment]
Yes, reoloading the integration does fix the speaker state issue.

**Comment by Strixx76** [comment]
Update: I have finnaly had some time to work on this integration, and I have found the bug that cased the issue with play and pause for DLNA. It was a typo in the underlying librarare that communicates with the speaker. It will be fixed in the next release and then you will also be able to see music info and music art when playing DLNA sources. The disconnection problem will also be fixed. I expect the release to be done before the end of the month.

**Comment by doemer** [comment]
Great to see this Issue was allready reported. I accidentially fell upon this Integration when I was searching for the known Commands to control my WAM350 and WAM750 speaker. 
Thx a lot for sharing your work!!!

**Comment by Strixx76** [comment]
Today I released a new version of the integration. I have had full focus on grouping and play media functions of the speakers for a long time. I thought this issue was resolved by an earlier fix and was about to close the issue when I did one final test. And i then realized that I have only solved the pause issue, and not the play part of the issue.

I think I have solved them both now, but need some more time for testing. But I will not release a new version before this integration is added to the HACS default repositories (or rejected).
